catalog/fixtures/init_promotion.yaml

```diff
@@ -1,40 +1,65 @@
 - model: catalog.promotion
```

Comment:
Change description to real description and move out date to different column 

Maybe separate to start date and end date column